lib/ansible/modules/network/routeros/routeros_command.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: routeros_command
+version_added: "2.7"
+author: "Egor Zaitsev (@heuels)"
+short_description: Run commands on remote devices running MikroTik RouterOS
+description:
+  - Sends arbitrary commands to an RouterOS node and returns the results
+    read from the device. This module includes an
+    argument that will cause the module to wait for a specific condition
+    before returning or timing out if the condition is not met.
+options:
+  commands:
+    description:
+      - List of commands to send to the remote RouterOS device over the
+        configured provider. The resulting output from the command
+        is returned. If the I(wait_for) argument is provided, the
+        module is not returned until the condition is satisfied or
+        the number of retries has expired.
+    required: true
```

Comment:
I *think* it is preferred to use `yes` and `no` in YAML documentation.

Reply:
You can actually use [almost whatever you want](http://yaml.org/type/bool.html). Here, I can see that both `required: true` and `require: yes` are used throughout Ansible but the former is used a lot more often. I think I'll keep it.

Reply:
I know, but I had the impression that there was a tendency to switch from `true`/`false` to `yes`/`no`. But maybe I'm also mistaken.

Reply:
Let's see if anyone else complains about that when reviewing ;)